apps/roam/package.json

```diff
@@ -25,6 +25,10 @@
     "widgets"
   ],
   "dependencies": {
+    "@emotion/react": "^11.14.0",
+    "@emotion/styled": "^11.14.0",
+    "@mui/icons-material": "^6.4.5",
+    "@mui/material": "^6.4.5",
```

Comment:
@sid597 Sorry, I should have been clear.  The link to the FAB was just as an example of the idea of what we wanted.  Let's stick to BlueprintJS for Roam and not import any new UI libraries.